frontend/src/components/Panel.svelte

```diff
@@ -40,7 +40,7 @@
             <ColorLegend {styles} />
           </div>       
             <div
-              class="mt-auto pointer-events-auto bg-base-200 shadow-md rounded-r-lg" 
+              class="mt-auto pointer-events-auto bg-base-200 shadow-md rounded-r-lg overflow-y-scroll" 
```

Comment:
While this might enable the scrolling feature, it also causes the previously mentioned bug on Windows to appear where the scroll bar is visible even if no scrolling is needed.

![image](https://user-images.githubusercontent.com/44969574/210446825-0ecac958-fdcd-48db-950b-ff22e614e7d2.png)

Reply:
I agree, but would you mind taking over? I cannot reproduce those bugs since they don't happen on Mac :x, would be very ineffective for me to try to fix something I cannot see

Reply:
I'll give it a go.